routers/web/repo/issue_list.go

```diff
@@ -459,6 +462,25 @@ func UpdateIssueStatus(ctx *context.Context) {
 	ctx.JSONOK()
 }
 
+func renderExclusiveLabelScopes(ctx *context.Context) {
+	labels, err := issues_model.GetLabelsByRepoID(ctx, ctx.Repo.Repository.ID, "", db.ListOptions{})
```

Comment:
If the repository is under an organization, maybe we also need to get the organization level labels.

Reply:
Good catch.  Updated.